concepts/inheritance/introduction.md

```diff
@@ -36,7 +37,7 @@ Now whenever we do,
 
 ```java
 Animal animal = new Lion(); //creating instance of Animal, of type Lion
-animal.bark();
+        animal.bark();
```

Comment:
What is the reason for adding this indentation? I don't think it makes the snippet more readable. Could you revert it please?